README.md

```diff
@@ -1,4 +1,33 @@
-# Twake server repository
+# Twake Chat Server
```

Comment:
Hi,
it's not really a Twake-Chat server, because Twake-Chat service has 2 main servers :
* the [Matrix Synapse](https://github.com/matrix-org/synapse/) server that provide the main Matrix implementation
* this server that provides:
  * local Matrix identity service
  * extensions to the Matrix protocol

Reply:
Hello!
The idea. was to make README files of all the TWP products more "user-friendly", easy to understand what is what

Reply:
Can we name it Twake Chat Server (local)?

Reply:
Or Twake Matrix extension server ?

Reply:
@guimard Ive committed new version. Can you please approve and merge?